aws/raw-to-ami.sh

```diff
@@ -31,7 +31,7 @@ pre_checks() {
 
 create_bucket() {
 	echo "Create s3 Bucket"
-	aws s3api create-bucket  --bucket ${BUCKET_NAME} --region ${REGION}
+	aws s3api create-bucket  --bucket ${BUCKET_NAME} --region ${REGION} --create-bucket-configuration LocationConstraint=${REGION}
```

Comment:
@snir911 looks like the `LocationConstraint` option doesn't work for `us-east-1`. So you'll need to use a conditional based on REGION

Reply:
Indeed, the docs at https://docs.aws.amazon.com/cli/latest/reference/s3api/create-bucket.html do not list us-east-1 as a valid value for LocationConstraint.